client/httpclient.go

```diff
@@ -33,6 +34,7 @@ func NewHTTP(logger types.Logger) *httpClient {
 	return w
 }
 
+// Start starts the client and runs until Stop is called.
```

Comment:
`Start()` is explained in more detail in the `OpAMPClient` interface description. We can either copy the comment from there or reference from here.

Reply:
Right, I wasn't really sure about this. What do you think about remaining ones `SetAgentDescription`, `SetHealth`? Should we copy for all of them?

Reply:
If you can add a reference in a way that it becomes a link in godoc [here](https://pkg.go.dev/github.com/open-telemetry/opamp-go) that would be best, so that we don't have to duplicate the content and risk diverging.

Reply:
Looks like there is no way to achieve this. I looked up how the packages in stdlib cross reference the functions from other modules. It just uses plain text like `os.Exit` or `json.Marshal` for the same reason.

Reply:
OK, let's just put plain text to reference `OpAMPClient.Start`.